profiles/base/package.use.mask

```diff
@@ -275,9 +275,11 @@ sys-devel/gcc vtv
 # invokes it wrong.
 dev-lang/moarvm asan
 
+# Jonathan Scruggs <j.scruggs@gmail.com> (28/09/2017)
```

Comment:
1. use the proper Gentoo date format
2. chuck out kensington's author line

Reply:
Okay. Sorry. I thought we had to keep credit to the first person that placed it there.

Reply:
Sorry for coming across as annoying if you want to keep the credit just use this format:
```
Some Gentoo Dev Who Will Merge This PR <someone@gentoo.org>
on behalf of Jonathan Scruggs <j.scruggs@gmail.com> (28 Sep 2017)
```

We do this to keep track of what's going on with these entries. Allowing "external" domain names in these files will confuse the hell out of developers and people after a while.

Reply:
Could you please update it accordingly? Thanks.

Reply:
Of course "Some Gentoo Dev" is an example. Use @kensington or me if you want. We will modify it on the fly before merging.

Reply:
@monsieurp @SoapGentoo I added that info like you asked. Sorry. Been really busy with work and knackered. :(